### PR TITLE
GH1446: Add SSL_SESSION_get0_cipher

### LINKED ISSUE #1446: OpenSSL 1.1.0 missing accessors
With v1.0.2 I can access that directly:

`SSL_SESSION->cipher`

In new version how can i access this?

### PR DESCRIPTION
**Pull request by richsalz**
Fixes #1446

### CODE REVIEW

File: doc/ssl/SSL_SESSION_get0_cipher.pod [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+L<ssl(3)>,
+L<d2i_SSL_SESSION(3)>,
+L<SSL_SESSION_get_time(3)>,
+L<SSL_SESSION_get0_hostname(3)>,
+L<SSL_SESSION_free(3)>
+
+=head1 HISTORY
+
+SSL_SESSION_get_protocol_version() was first added to OpenSSL 1.1.0
```
Comment (oneton) [inline doc/ssl/SSL_SESSION_get0_cipher.pod]: Minor detail: I guess this history section should refer to the new method?

Comment (richsalz) [inline doc/ssl/SSL_SESSION_get0_cipher.pod]: Yes, thanks, fixed.  Updated commit pushed.

File: doc/ssl/SSL_SESSION_get0_cipher.pod [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+=head1 NAME
+
+SSL_SESSION_get0_cipher - retrieve the SSL cipher associated with a session
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ const SSL_CIPHER *SSL_SESSION_get0_cipher(const SSL_SESSSION *s);
```
Comment (mattcaswell) [inline doc/ssl/SSL_SESSION_get0_cipher.pod]: Typo in the parameter type.

**Comment by mattcaswell** [comment]
+1 assuming you fix the typo.

File: doc/ssl/SSL_SESSION_get0_cipher.pod (hunk 2 above)
Comment (richsalz) [inline doc/ssl/SSL_SESSION_get0_cipher.pod]: fixed, tnx.  will merge.

**Comment by richsalz** [comment]
commit e928132 merged.

**Pull request closed without merging**